gcp/gke.ts

```diff
@@ -52,9 +52,9 @@ export class GcpCloud {
 
     // separate node pool
     const nodePool = new gcp.container.NodePool(
-      util.name("primary-node-pool"),
+      util.name("primary-np"),
```

Comment:
NodePools names have a 40 chars limit. Better to cut off some chars here, then.

Reply:
Add that as a code comment, please. Eventually, we'll do the reverse edit and shoot ourselves in the foot.

Reply:
There's a handful of limitations when it comes to length of names of things. These limitations are arbitrary and cloud dependant (impossible to remember them all).